polytope/polytope.py

```diff
@@ -78,7 +78,10 @@
     logger.warn(
         '`polytope` failed to import `cvxopt.glpk`.\n'
         'Will use `scipy.optimize.linprog`.')
-
+try:
+    xrange
```

Comment:
I am not sure about efficiency, but this line is executed once per module import. The `try`-`except` seems more pythonic and readable.